### DISCUSSION TITLE
Remove key function?

### DISCUSSION THREAD

**Discussion by Tosche** [nickcoutsos/keymap-editor#185] [Q&A]
I couldn't find a way to remove a function assigned to a key (kp for example). How do I do that? And can you add that as a button in the behaviour dialog?

**Comment by nickcoutsos** [comment]
That sounds like you want the [`&trans` and `&none` behaviors](https://zmk.dev/docs/behaviors/misc)?

**Reply by Tosche** [reply]
That's so true, duh! Sorry I'm new to the whole ZMK experience.

**Reply by nickcoutsos** [reply]
No worries!
